assets/airports/rjss.json

```diff
@@ -0,0 +1,960 @@
+                ["N38d08m19.58", "E140d53m55.40", "11.5ft"],
+                ["N38d08m32.18", "E140d55m57.56", "15.1ft"]
+            ],
+            "ils": [false, true],
+            "ils_distance": [27, 27],
+            "loc_maxDist": [27, 27],
+            "glideslope": [3.00, 3.00]
+        },
+        {
+            "name": ["12", "30"],
+            "end": [
+                ["N38d08m22.05", "E140d54m53.09", "6ft"],
+                ["N38d08m03.96", "E140d55m36.72", "5ft"]
+            ],
+            "ils": [false, false]
+        }
+    ],
+    "airways": {
+        "L512": ["GTC", "TATAM", "IGOBI", "KAMSA", "ANDOL"],
+        "R211": ["AVGOK", "NISIN", "KADBO", "TAVIS", "CAYAH", "BASIN", "GTC", "IXE", "GLAXY", "SWAMP"],
+        "R217": ["NODAN", "ASTER", "SDE", "FINCH", "YONEX", "ELDAK", "GTC"],
+        "V10" : ["GTC", "WAPPA", "SAMBO", "MIS"],
+        "V15" : ["GTC", "KALON", "AKAGI", "SYE"],
+        "V22" : ["CHE", "TOBBY", "OBCHI", "CAVES", "MQE", "SDE", "HERON", "FKE"],
+        "V30" : ["CHE", "ESANN", "HWE", "TSURU", "OBAKO", "ATUMI", "BUDOH", "INAHO", "GTC", "TERAD", "HAKBA", "GORYU", "IKUJI", "URUSI", "IMIZU", "OYABE", "KMC"],
+        "V31" : ["HWE", "HIBAR", "BYOBU", "MRE", "MAGGY", "UWE", "YAYOI", "KENSI", "AWASI", "GTC", "MAGNA", "NAEBA", "AZUMI", "OWARI", "KCC"],
+        "V33" : ["MIS", "HPE", "SDE"],
+        "V34" : ["MQE", "MORIO", "HPE", "HANKA", "MAMRO", "GTC"],
+        "Y102": ["CAVES", "HPE", "METEL", "SDE", "HERON", "FKE"],
+        "Y106": ["MQE", "SDE", "RUBIS"],
+        "Y12" : ["CHE", "HWE", "TAPPI", "ARIKA", "OTOME", "GTC", "FINGA", "MBE", "TOLUM", "NAKTU", "KOHWA", "SHIMA", "AYANO", "RAYJO", "KEC", "LAXEL"],
+        "Y122": ["HWE", "OBAKO", "YOHKO", "ATUMI", "BUDOH", "INAHO", "GTC"],
+        "Y124": ["MQE", "HPE", "TOLEK", "HANKA", "MAMRO", "GTC"],
+        "Y142": ["GTC", "NESKO", "SAMON"],
+        "Y15" : ["TENRU", "MYOGI", "TADMI", "DANDY", "HEIKE", "INAWA", "SHIRO", "NIRAX", "OWLET"],
+        "Y303": ["GTC", "VIKET", "AKTIL", "SASAP", "ESKAS", "ENTAK"],
+        "Y312": ["KCC", "AZUMI", "PAPNA", "NAEBA", "MAGNA", "GTC", "KENSI", "YAYOI", "UWE"],
+        "Y347": ["IGROD", "EKVIK", "HANAB", "GOLDO", "GTC"],
+        "Y45":  ["GTC", "TERAD", "HAKBA", "GORYU", "IKUJI", "URUSI", "IMIZU", "OYABE", "KMC", "SONBU", "MADUL", "SAKYU", "MIHOU", "KYOKA", "CARPS", "MARCO", "SQRUM", "YANAI", "OOITA", "SOBOH", "RAFUL", "MOCOS", "HKC", "ISKID", "MAKRA", "SAZMA", "BOMAP", "ANOXA", "MEKAX", "HACHA", "ONC", "LEXUS", "OKUMA", "NHC"],
+        "Y512": ["ADNAP", "OATIS", "BEKEN", "SDE", "ELDAK", "GTC"],
+        "Y514": ["ASTER", "SDE"],
+        "Y515": ["KMC", "ZUWAI", "ETIGO", "VIKET", "AIZUH", "LANCE"],
+        "Z119": ["GTC", "YANAT", "FKE"]
+    },
+    "sids": {
+        "CUBIC3": {
+            "icao": "CUBIC3",
+            "name": "Cubic Three",
+            "rwy": {
+                "RJSS09": [["_RWY2702DME", "A5+"], "SS901"],
+                "RJSS27": [["_RWY0902DME", "A5+"], "BUBLE"]
```

Comment:
`_RWY0902DME` _might_ need to be made a fly-over fix to ensure aircraft don't begin a turn before the end of the runway